project-base/storefront/components/Pages/Search/SearchContent.tsx

```diff
@@ -41,21 +41,21 @@ export const SearchContent: FC<SearchContentProps> = ({ searchResults, fetching
                         {!!searchResults.articlesSearch.length && (
                             <div className="mt-6">
                                 <div className="h3 mb-3">{t('Found articles')}</div>
-                                <SimpleNavigation linkType="blogArticle" listedItems={searchResults.articlesSearch} />
+                                <SimpleNavigation listedItems={searchResults.articlesSearch} />
```

Comment:
Explain this to me. Does this fix some problem, because inside e.g. articlesSearch can be a link with a different type? Or what exactly does this solve (I couldn't find any description of the issue/motivation in commit message/upgrade notes)?

Reply:
Explained on the Slack call